### ISSUE TITLE
[bluetooth/bthecho] Unable to build bthecho

### ISSUE THREAD

**Issue by steam3d** [microsoft/Windows-driver-samples#1177]
When build the bthecho sample as described in README I get error
```
'BthEchoSampleSrv' is reserved
'BthEchoSampleCli' is reserved
```

![image](https://github.com/microsoft/Windows-driver-samples/assets/30986874/97493110-a0b7-41c3-8081-7a861cf768e8)

# BthEchoSampleCli.inx file:
 
https://github.com/microsoft/Windows-driver-samples/blob/0038767e5a3f27d464d20f2d4b97f607e58aac5d/bluetooth/bthecho/bthcli/sys/BthEchoSampleCli.inx#L56

https://github.com/microsoft/Windows-driver-samples/blob/0038767e5a3f27d464d20f2d4b97f607e58aac5d/bluetooth/bthecho/bthcli/sys/BthEchoSampleCli.inx#L67

# BthEchoSampleSvr.inx file:

https://github.com/microsoft/Windows-driver-samples/blob/0038767e5a3f27d464d20f2d4b97f607e58aac5d/bluetooth/bthecho/bthsrv/sys/BthEchoSampleSrv.inx#L56

https://github.com/microsoft/Windows-driver-samples/blob/0038767e5a3f27d464d20f2d4b97f607e58aac5d/bluetooth/bthecho/bthsrv/sys/BthEchoSampleSrv.inx#L67

**Comment by NeoAdonis** [comment]
This is by design: we expect samples to be changed and used only as the basis to build new drivers, rather than being built as they are.

There are ways to exclude this error. We use those, for example, to validate that the sample driver builds as expected. However, the right way to resolve this issue is to rename the service.

I'm closing this for now but feel free to add more comments if you have any other questions or issues!

**Comment by steam3d** [comment]
@NeoAdonis I quote README file:
![image](https://github.com/microsoft/Windows-driver-samples/assets/30986874/d32b3aae-0f17-4a11-bc68-20c3d5e8af43)

I expect to get a successful build according to the REAMDE file, but not an error. This should definitely be fixed so as not to confuse developers and generate additional issues

**Comment by NeoAdonis** [comment]
@steam3d I agree. Thanks for letting me know! I'll reach out to the team to see what's the best approach to document this.

**Comment by v-junyli** [comment]
@steam3d I took a look but I lack the context in between, I tried building the bthecho sample with the latest EWDK, it's successful. I think it's fixed at some point last year. Can you confirm that it's same for you?

**Comment by v-junyli** [comment]
we're closing this item as we haven't heard back for a month, feel free to comment to reactivate if the issue is still relevant.
